.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
+api/bitcointranscripts
```

Comment:
what is this?

Reply:
these were some test files for PR opening. i've removed the line in the latest commit together with some redundant comments